### PR TITLE
Add artefact tripe ration Liver of Ahriman

### PR DESCRIPTION
**Pull request by ars3niy**
Not randomly generated, generated in Thoth Amon's inventory or can be wished for by non-Barbarians.

An infinite food source which, however, restores itself slower than its nutrition lasts. Invoke for confusion.

### CODE REVIEW

File: src/makemon.c [hunk 1]
```diff
@@ -714,6 +714,24 @@ m_initinv(struct monst *mtmp)
...
+            if (!exist_artifact(TRIPE_RATION, artiname(ART_LIVER_OF_AHRIMAN))) {
+                otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+                otmp = oname(otmp, artiname(ART_LIVER_OF_AHRIMAN), ONAME_LEVEL_DEF);
+                (void) mpickobj(mtmp, otmp);
+            }
+            otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+            otmp = oname(otmp, "Stomach of Arihman", ONAME_NO_FLAGS);
+            (void) mpickobj(mtmp, otmp);
+            otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+            otmp = oname(otmp, "Small intestine of Arihman", ONAME_NO_FLAGS);
```
Comment (mark-mcknight-oncorps) [inline src/makemon.c]: Small typo, should be: "Small intestine of Ahriman"

File: src/makemon.c [hunk 2]
```diff
@@ -714,6 +714,24 @@ m_initinv(struct monst *mtmp)
...
+            (void) mpickobj(mtmp, otmp);
+            otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+            otmp = oname(otmp, "Big intestine of Arihman", ONAME_NO_FLAGS);
```
Comment (mark-mcknight-oncorps) [inline src/makemon.c]: "Large intestine of Ahriman"

File: src/makemon.c [hunk 3]
```diff
@@ -714,6 +714,24 @@ m_initinv(struct monst *mtmp)
...
+            (void) mpickobj(mtmp, otmp);
+            otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+            otmp = oname(otmp, "Gall bladder of Arihman", ONAME_NO_FLAGS);
```
Comment (mark-mcknight-oncorps) [inline src/makemon.c]: "Gallbladder of Ahriman"

File: src/makemon.c [hunk 4]
```diff
@@ -714,6 +714,24 @@ m_initinv(struct monst *mtmp)
...
         } else if (quest_mon_represents_role(ptr, PM_MONK)) {
             (void) mongets(mtmp, rn2(11) ? ROBE : CLOAK_OF_MAGIC_RESISTANCE);
+        } else if (ptr == &mons[PM_THOTH_AMON]) {
+            if (!exist_artifact(TRIPE_RATION, artiname(ART_LIVER_OF_AHRIMAN))) {
+                otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+                otmp = oname(otmp, artiname(ART_LIVER_OF_AHRIMAN), ONAME_LEVEL_DEF);
+                (void) mpickobj(mtmp, otmp);
+            }
+            otmp = mksobj(TRIPE_RATION, FALSE, FALSE);
+            otmp = oname(otmp, "Stomach of Arihman", ONAME_NO_FLAGS);
```
Comment (mark-mcknight-oncorps) [inline src/makemon.c]: Ahriman is misspelled, should be: "Stomach of Ahriman"

**Review by mark-mcknight-oncorps — CHANGES_REQUESTED** [review]
This looks great! The code is very clean, great job. There are a few small typos, other than that it's good, let's add it!

File: src/makemon.c (hunk 1 above)
Comment (ars3niy) [reply src/makemon.c]: should all be fixed now

**Pull request merged**